src/test/java/seedu/address/testutil/TypicalContacts.java

```diff
@@ -149,32 +87,20 @@ public class TypicalContacts {
             .build();
 
     // Manually added
-    public static final Contact HOON = new ContactBuilder()
-            .withName("Hoon Meier")
-            .withId("test_8-123")
-            .withPhone("8482424")
-            .withEmail("stefan@example.com")
-            .withAddress("little india")
+    public static final Organization TIKTOK = new OrganizationBuilder()
```

Comment:
Not really sure if this way of testing is ideal but I will close one eye since there's the cleanup phase

Reply:
The one where we declare `VALID_NAME` sth like that

Reply:
With the linking between organization and recruiter, its a bit messy to test but yeah should look into cleaning things up and making sure the dependencies for each test do not mess things up real bad.